full-service/src/service/sync.rs

```diff
@@ -30,6 +30,10 @@ use mc_transaction_core::{
 };
 use rayon::prelude::*;
 
+use reqwest::{
+    blocking::Client, // FIXME: if we don't want this to be blocking, we'll be awaiting futures
```

Comment:
⚠️ Note - currently implemented with blocking client. When not using blocking client, everything needs to move to async. I wasn't ready to evaluate and make that change

Reply:
agreed that we should be leery of full-service having its core scanning loop blocked by a web get or post to a webhook.

Reply:
Moved the webhook into its own WebhookThread - we may want to use this as a touchpoint to evaluate the async and whether we're happy with the concurrency model we're using for account sync as well